capstone/src/main/java/com/google/sps/servlets/AnnouncementsServlet.java

```diff
@@ -12,18 +25,53 @@
 @WebServlet("/announcements")
 public class AnnouncementsServlet extends HttpServlet {
 
+  private static final String CLUB_NAME_PROP = "name";
```

Comment:
Might be good to put these in the Constants file too? I know that I'll be using "name" as a constant.

Reply:
Wait "name" is already a constant in Constants.java. Is this one different?

Reply:
Nah, I think I just missed that